vision/build.gradle

```diff
@@ -9,92 +9,146 @@ plugins {
 	id "com.peterabeles.gversion" version "1.10"
 	id "com.diffplug.spotless" version "6.24.0"
 	id "com.vanniktech.maven.publish" version "0.30.0"
+	id "edu.wpi.first.GradleRIO" version "2025.1.1-beta-2"
 }
 
 java {
 	sourceCompatibility = JavaVersion.VERSION_17
 	targetCompatibility = JavaVersion.VERSION_17
 }
 
+def ROBOT_MAIN_CLASS = "coppercore.vision.VisionLocalizer"
```

Comment:
since we are trying gradlerio, can you open a PR to show this works in 2025-Robot-Code?

You could publish an alpha release (something like 2025.0.0-alpha) and pull it from maven central to prove the concept